contracts/src/TroveManager.sol

```diff
@@ -769,6 +796,7 @@ contract TroveManager is LiquityBase, ITroveManager, ITroveEvents {
 
             remainingBold -= singleRedemption.boldLot;
             singleRedemption.troveId = nextUserToCheck;
+            singleRedemption.isZombieTrove = false;
```

Comment:
We only need to do this if it was a `zombieTrove`. Though seems fine to just do it here at the end.